saleor/graphql/product/sorters.py

```diff
@@ -144,9 +144,9 @@ def description(self):
             ProductOrderField.NAME.name: "name",
             ProductOrderField.PRICE.name: "price",
             ProductOrderField.TYPE.name: "type",
-            ProductOrderField.MINIMAL_PRICE.name: (
-                "a minimal price of a product's variant"
-            ),
+            # ProductOrderField.MINIMAL_PRICE.name: (
+            #     "a minimal price of a product's variant"
+            # ),
```

Comment:
I guess this field should be renamed to `DISCOUNTED_PRICE`. If we change it we need to open a task for storefront because they're using `MINIMAL_PRICE` now if I'm correct.

Reply:
We will handle it in separate [task](https://app.clickup.com/t/2549495/SALEOR-655). I left the comment in the code.